# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/mp4_demuxer/TestMP4.cpp
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd dom/media/gtest/mp4_demuxer/TestMP4.cpp
--- a/dom/media/gtest/mp4_demuxer/TestMP4.cpp	Fri Jan 27 20:35:34 2023 +0000
+++ b/dom/media/gtest/mp4_demuxer/TestMP4.cpp	Fri Jan 27 22:54:08 2023 +0200
@@ -92,12 +92,12 @@
   io = {vector_reader, &buf};
   rv = mp4parse_new(&io, &parser);
   ASSERT_EQ(parser, nullptr);
-  EXPECT_EQ(rv, MP4PARSE_STATUS_MOOV_MISSING);
+  EXPECT_EQ(rv, MP4PARSE_STATUS_NO_MOOV);
 
   buf.buffer.reserve(4097);
   rv = mp4parse_new(&io, &parser);
   ASSERT_EQ(parser, nullptr);
-  EXPECT_EQ(rv, MP4PARSE_STATUS_MOOV_MISSING);
+  EXPECT_EQ(rv, MP4PARSE_STATUS_NO_MOOV);
 
   // Empty buffers should fail.
   buf.buffer.resize(4097, 0);